_posts/2023-04-18-ai-based-assistant.md

```diff
@@ -0,0 +1,55 @@
+
+***Generative AI can help people find the right way to express themselves***
+
+This Beta tool was developed as a joint project between Hubert Burda Media and HolidayCheck Group. [Jean-Paul Schmetz](https://de.linkedin.com/in/jpschmetz), Chief Scientist at Hubert Burda Media and [Konark Modi](https://de.linkedin.com/in/konarkmodi), Vice President Technology at HolidayCheck are leading the development of this AI assistant and answer the most important questions here. Jean-Paul Schmetz explains how AI can help with writing reviews: “Just like a calculator can help people make sure they calculate correctly, Generative AI can help people find the right way to express themselves. We have known for years that most of our users are somewhat insecure about their writing skills, and it prevents them from sharing their experiences with the rest of our users. While a lot of people focus on the ability to generate fake content with Generative AI, we focus on its ability to assist users in writing reviews that reflect their experience”, says Jean-Paul Schmetz.
+
+We asked both experts for more information about the HolidayCheck review assistant. Here are their answers:
+
+***Why did you develop your own AI assistant for HolidayCheck?***
+
+We strive to reduce the barrier to entry for users wanting to write a review. One does not need to be an expert in any language to submit a review, nor does one have to spend hours finding the words to use or not use. The users can simply start speaking or typing and the assistant helps to write a proper narrative with just a few words.
+
+***Is it also possible to upload pictures?***
+
+In the beta phase we are only collecting two types of content: Text and audio. If you wish to submit pictures, they can still be uploaded via our regular system: https://www.holidaycheck.de/wcf
+
+***Where can I give feedback directly?***
+
+We are looking forward to your feedback! You can give your feedback directly either by using the Review Assistant or by clicking the “Feedback” button on the right side of the page.
+
+***How long will it be before I can see the finished feature?***
+
+We are still working on the technology and on the features. We plan to launch it later this year. For more information, please sign up to our newsletter: https://www.holidaycheck.de/newsletter
+
+***Do we risk diminishing the value of genuine reviews vs AI written reviews?***
+
+This is a fair question and one we ought to ask ourselves. However, reviews written by this assistant will still go through all our strict checks and balances. We would not start by publishing them directly, but by carefully moderating them. We must keep in mind that it’s still a review. The users can still use assistants outside of HolidayCheck, so we should accept this and embrace this technology trend – without diluting the brand value and authority.
```

Comment:
This is a fair question we ought to ask ourselves. However ...

Reply:
This is slightly complicated to ask. Do we mean,
"This is a fair question we ought to ask ourselves. Users, even today, can use assistants outside of HolidayCheck to write reviews and we should definitely embrace the technology trend. However, all reviews submitted to HC, including ones generated by this assistant go through all our strict checks and balances, thus not diluting the brand value and authenticity."